1-JavaScript/Week2/homework/ex7-mindPrivacy.js

```diff
@@ -30,8 +30,11 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function filterPrivateData(data) {
+  return data.map((element) => {
+    const { name, occupation, email } = element;
+    return { name, occupation, email };
+  });
```

Comment:
Amazing 🤩 

One tip (you don't have to change the code, it's good already): you can also destructure directly after `map`. Again, you don't have to change it but just wanted to provide some more information 🙂 I like how you explicitly destructure the elements inside the function, makes it a bit clearer (especially if you have to destructure a lot of elements).

```suggestion
function filterPrivateData(data) {
  return data.map(({ name, occupation, email ) => {
    return { name, occupation, email };
  });
```